README.md

```diff
@@ -53,12 +53,28 @@ doctrine:
 
 A full list of all possible options can be found here: http://php.net/manual/en/function.curl-setopt.php
 
+By default, ```UPDATE queries are converted to PUT to work with the majority of
```

Comment:
The closing ``` are missing. And if you surround UPDATE with ``` you should also surround the word "PUT". Text itself is very clear and detailed. Good job